### ISSUE TITLE
Adjust / reduce the rate that text is output to the screen for visual effect.

### ISSUE THREAD

**Issue by koblue** [microsoft/terminal#11182]
-->

Slow the output of text to each line, replicating the slow modems of old, teletypes and sci-fi movie computer text.

Cathode, a now dead/obsolete fancy terminal for MacOS (similar to CoolRetroTerm) had a similar feature. 

# Proposed technical implementation details (optional)

Artificially reduce the speed new characters are output to a line in any given terminal session.

**Comment by zadjii-msft** [comment]
While this does seem like a fun gimmick, couldn't this be accomplished with some sort of shim executable like `slow-output-down.exe` that just starts some other commandline and buffers the output itself? I'm not totally sure that this is something that needs to be built in to the Terminal...

**Comment by j4james** [comment]
> While this does seem like a fun gimmick, couldn't this be accomplished with some sort of shim executable like `slow-output-down.exe` that just starts some other commandline and buffers the output itself?

Yeah, I have a little python script I use for viewing old VT animations. That said, I did also think it might be neat to have it built into the terminal as a hidden feature which could possibly be triggered with a [`DECSCS`](https://vt100.net/docs/vt510-rm/DECSCS.html) sequence. I don't feel that strongly about it, though, because it's honestly not the sort of thing I'd use very often.

**Comment by koblue** [comment]
I agree this is 95% for show, but I really did find it made working with big lists in the command line much easier.  Thought I'd float it just in case.

I've found some python scripts that replicate the effect and while not remotely sane to pipe every terminal command to it just for eye candy, since what I'm mostly dealing with is from my own scripts I can just tack on some code to them instead.
